src/server/index.js

```diff
@@ -76,8 +99,10 @@ function afterLoaded(asset, { config: { tce } }) {
   const { sproutVideoApiKey: apiKey } = tce;
   const client = createClient({ apiKey });
   return client.videos.get(videoId)
-    .then(({ embed_code: embedCode }) => {
+    .then(({ embedCode, selectedPosterFrameNumber, assets }) => {
```

Comment:
Can we rename `selectedPosterFrameNumber` to `selectedPosterFrameIndex` to stay consistent?